cpp/log/consistent_store.h

```diff
@@ -79,6 +81,10 @@ class EntryHandle : public EntryHandleBase {
     return &entry_;
   }
 
+  bool SerializeToString(std::string* output) const override {
```

Comment:
This is to implement the pure virtual in the parent class, which allows method that only need to write the entry to etcd (without needing anything else from the entry) to take the non-templated parent class (avoiding the need to be templated themselves).